index.js

```diff
@@ -120,3 +151,12 @@ console.log('hello world')
  */
 
 // TODO: ADD CODE HERE
+const multiplyByTwo = function(number){
+    return number * 2;
+}
+const num1 = multiplyByTwo(1);
+const num2 = multiplyByTwo(5);
+const num3 = multiplyByTwo(10);
+console.log(num1);
+console.log(num2);
+console.log(num3);
```

Comment:
Looks great!
You could also if you wanted log these all out on only line by writing the console log with multiple arguments, such as
`console.log (num1, num2, num3)`